lib/internal/Magento/Framework/App/Test/Unit/Config/MetadataConfigTypeProcessorTest.php

```diff
@@ -11,7 +11,7 @@
 use Magento\Framework\App\Config\Data\ProcessorInterface;
 use Magento\Framework\App\Config\Initial;
 use Magento\Framework\App\Config\MetadataConfigTypeProcessor;
-use \PHPUnit_Framework_MockObject_MockObject as MockObject;
+use \PHPUnit\Framework\MockObject\MockObject as MockObject;
```

Comment:
```suggestion
use \PHPUnit\Framework\MockObject\MockObject;
```
Could you please also remove as where this is not necessary ?